srcjs/shinyapp.js

```diff
@@ -310,6 +310,10 @@ var ShinyApp = function() {
       var show = condFunc(scope);
       var showing = el.css("display") !== "none";
       if (show !== showing) {
+        el.trigger({
+          type: 'shiny:conditional',
```

Comment:
I have added changes according to the last three comments. For this one, I don't quite get it. I can certainly add the `$(document).trigger()` call, but I don't see how it can be useful.